lib/FileModel.php

```diff
@@ -34,7 +37,30 @@ public function determineFileMimeTypesInfo($currentDirectory) {
             }
         }
     }
-
+    /**
+     * @param Folder $currentDirectory
+     */
+    public function determineBiggestFiles($currentDirectory) {

```

Comment:
As i understood, you just sorting all the files, but it can be problem for big folder.  You should add N as a parameter and return only biggest N file. N will given by user.

Reply:
I will work on this after I merge biggest n file function and mime type function